brainstorm/package.json

```
@@ -45,10 +45,11 @@
   },
   "dependencies": {
     "@fluentui/react": "^8.71.0",
-    "fluid-framework": "^0.48.1",
-    "@fluidframework/azure-client": "^0.48.1",
-    "@fluidframework/test-client-utils": "^0.48.1",
+    "@fluidframework/azure-client": "^0.59.3000",
+    "@fluidframework/container-loader": "^0.59.4001",
```

should these be consistently update to 1.0.0?

---

That is what I would think but azure-client package seems outdated. Whenever I tried to update fluid-framework to 1.0.0, azure-client stayed behind and was causing incompatibility issues

---

azure-client is not yet released. These should be on the same version though.

```suggestion
    "@fluidframework/container-loader": "~0.59.3000",
```

---

removing container-loader